src/witness/witness_calculator.rs

```diff
@@ -162,6 +212,22 @@ mod runtime {
         Function::new_native(store, func)
     }
 
+    // Circom 2.0
+    pub fn exception_handler(store: &Store) -> Function {
```

Comment:
Worth feature-gating these to keep the lib small? `#[cfg(feature = "circom-2)]`

Reply:
Done.